docker-compose.yaml

```diff
@@ -1,34 +1,84 @@
 
-version: "3.8"
-
+version: "3"
 services:
-  # account:
-  #   build: "src/account/"
-  #   command: "npm run account"
-  #   ports: 
-  #     - "5000:5000" 
-  #   env_file: "src/account/.env"
-  #   dns:
-  #     - "8.8.8.8"
-  models:
-    build: "src/models/"
-    command: "echo Models Container Working"
-  routes:
-    build: "src/routes/"
-    command: "npx ts-node github.ts"
+  activemq: 
+    image: rmohr/activemq
+    container_name: activemq
+    networks:
+      - backend
+    ports: 
+      - "8161:8161"
+      - "1883:1883"
+    env_file: ".env"
+
+  # server:
+  #   build: "."
+  #   command: "npm run server"
+  #   ports:
+  #     - "4000:4000"
+  #   networks:
+  #     - backend
+  #   env_file: ".env"
+  #   depends_on:
+  #     - "activemq"
+  account:
+    container_name: "account"
+    build: "src/account/"
+    command: "npm run account"
+    ports: 
+      - "5000:5000" 
+    env_file: ".env"
+    dns:
+      - "8.8.8.8"
+    networks:
+      - backend
+  publish:

```

Comment:
@KevinLemon112 The publish routes weren't working on port 4000, but they work now.  You can test by using Postman or a local browser and going to http://localhost:4000/publish/all

You should see something like this:
```json
{
    "id": "all",
    "message": "From Publish Service"
}
```